src/v/serde/serde.h

```diff
@@ -157,6 +157,13 @@ struct is_absl_node_hash_map<absl::node_hash_map<Args...>> : std::true_type {};
 template<typename T>
 inline constexpr bool is_absl_node_hash_map_v = is_absl_node_hash_map<T>::value;
 
+template<typename T>
+struct is_duration : std::false_type {};
+template<typename... Args>
+struct is_duration<std::chrono::duration<Args...>> : std::true_type {};
+template<typename T>
+inline constexpr bool is_duration_v = is_duration<T>::value;
```

Comment:
I copy pasted for consistency. Is it ok to define just this one with concepts? (double checking before I redo this).

Reply:
I'm like 2/10 on this. If it works, it works. This is the context: https://github.com/redpanda-data/redpanda/pull/5288

Reply:
Can't we just use the builtin std::same_as? This seems to work. LMK if you think otherwise.

```
template<class T>
concept is_duration_v = std::same_as<T, std::chrono::duration<typename T::rep, typename T::period>>;
```

Reply:
Yeah, that works. The `_v` is unusual for a concept, and I think `chrono` should be in the name.